R/text_to_date.R

```diff
@@ -0,0 +1,132 @@
+                              |[:digit:]{2}-[:digit:]{1}-[:digit:]{2}|
+                              |[:digit:]{1}-[:digit:]{1}-[:digit:]{2}|
+                              |[:digit:]{4}-[:digit:]{2}-[:digit:]{2}|
+                              |[:digit:]{4}-[:digit:]{2}-[:digit:]{1}|
+                              |[:digit:]{4}-[:digit:]{1}-[:digit:]{2}|
+                              |[:digit:]{4}-[:digit:]{1}-[:digit:]{1}|
+                              |[:digit:]{2}/[:digit:]{2}/[:digit:]{4}|
+                              |[:digit:]{1}/[:digit:]{2}/[:digit:]{4}|
+                              |[:digit:]{2}/[:digit:]{2}/[:digit:]{2}|
+                              |[:digit:]{1}/[:digit:]{2}/[:digit:]{2}|
+                              |[:digit:]{2}/[:digit:]{1}/[:digit:]{4}|
+                              |[:digit:]{1}/[:digit:]{1}/[:digit:]{4}|
+                              |[:digit:]{2}/[:digit:]{1}/[:digit:]{2}|
+                              |[:digit:]{1}/[:digit:]{1}/[:digit:]{2}|
+                              |[:digit:]{4}/[:digit:]{2}/[:digit:]{2}|
+                              |[:digit:]{4}/[:digit:]{2}/[:digit:]{1}|
+                              |[:digit:]{4}/[:digit:]{1}/[:digit:]{2}|
+                              |[:digit:]{4}/[:digit:]{1}/[:digit:]{1}|
+                              |[:digit:]{3}\\s[:alpha:]{4}\\s[:digit:]{2}|
+                              |[:digit:]{4}\\s[:alpha:]{4}\\s[:digit:]{2}|
+                              |[:digit:]{4}\\s[:alpha:]{5}\\s[:digit:]{2}|
+                              |[:digit:]{4}\\s[:alpha:]{5}\\s[:digit:]{2}|
+                              |[:digit:]{4}\\s[:alpha:]{6}\\s[:digit:]{2}|
+                              |[:digit:]{4}\\s[:alpha:]{7}\\s[:digit:]{2}|
+                              |[:digit:]{4}\\s[:alpha:]{8}\\s[:digit:]{2}|
+                              |[:digit:]{4}\\s[:alpha:]{9}\\s[:digit:]{2}|
+                              |[:digit:]{3}\\s[:alpha:]{4}\\s[:digit:]{1}|
+                              |[:digit:]{4}\\s[:alpha:]{4}\\s[:digit:]{1}|
+                              |[:digit:]{4}\\s[:alpha:]{5}\\s[:digit:]{1}|
+                              |[:digit:]{4}\\s[:alpha:]{5}\\s[:digit:]{1}|
+                              |[:digit:]{4}\\s[:alpha:]{6}\\s[:digit:]{1}|
+                              |[:digit:]{4}\\s[:alpha:]{7}\\s[:digit:]{1}|
+                              |[:digit:]{4}\\s[:alpha:]{8}\\s[:digit:]{1}|
+                              |[:digit:]{4}\\s[:alpha:]{9}\\s[:digit:]{1}")
+  out
+}
+
+re_order <- function(l) {
+  l <- stringr::str_squish(l)
+  st <- stringr::word(l, 1)
+  mi <- stringr::word(l, 2)
+  ed <- stringr::word(l, 3)
+  out <- ifelse(stringr::str_starts(l, "[:digit:]{4}"),
+                paste0(ed, "-", mi, "-", st), l)
+  out <- ifelse(stringr::str_starts(out, "[:alpha:]"),
+                paste0(mi, "-", st, "-", ed), out)
+  out <- stringr::str_remove_all(out, "-NA|NA-|NA")
+  out
+}
```

Comment:
Is this `resequence()`?

Reply:
This is not `resequence()`, was just a helper function to re-order date components. This has been removed now that the text functionality has been incorporated into `as_messydate()`.